tasks/app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<h1>Listing tasks</h1>
+
+<table>
+  <tr>
+    <th>Name</th>
+    <th>Description</th>
+    <th></th>
+    <th></th>
+    <th></th>
+  </tr>
+
+<% @tasks.each do |task| %>
+  <tr>
+    <td><%= task["name"] %></td>
+    <td><%= task["description"] %></td>
+    <td><%= link_to 'Show', task_path(task["id"]) %></td>
+    <td><%= link_to 'Edit', edit_task_path(task["id"]) %></td>
+    <td><%= link_to 'Destroy', task_path(task["id"]), method: :delete, data: { confirm: 'Are you sure?' } %></td>
```

Comment:
You should be able to get task attributes using normal methods `task.id / task.description / task.name`
Any reason for using Hash-like syntax?